README.md

```diff
@@ -115,6 +115,9 @@ customresourcedefinition.apiextensions.k8s.io/warmimages.mattmoor.io configured
 `ko` has four commands, most of which build and publish images as part of
 their execution.  By default, `ko` publishes images to a Docker Registry
 specified via `KO_DOCKER_REPO`.
+You'll need to be authenticated with your `KO_DOCKER_REPO` before pushing
```

Comment:
nit:  I'd add a **Note:** and an empty line above this new sentence

Reply:
You got it.